docs/css/custom.css

```diff
@@ -1112,11 +1112,13 @@ table {
   width: 100%;
   overflow-wrap: normal;
   border-collapse: collapse; /* Ensures that the borders collapse into a single border */
+  word-break: normal;
```

Comment:
If you add `overflow-x: auto;` here, does it solve the problem without needing all the new `div`s?

Reply:
This makes sense to me, let me add it to somewhere else